### ISSUE TITLE
[BUG] Application with empty feed and crashes in 0.81.3

### ISSUE THREAD

**Issue by gzuuus** [vitorpamplona/amethyst#703]
**Describe the bug**
Amethyst has stopped working today, yesterday I updated to the new version 0.81.3 and it seemed to work fine, today using it seemed to work fine, but now it has stopped working, it doesn't show the relays that I have set in my profile and it stays all the time in the 'Feed is empty' screen, I have tried to clear the cache of the application, but it hasn't worked. I also tried to clean all the data of the application, and even reinstall it, this produced that the application now crashes when I try to log in with my private key. For me it has become unusable for the moment

**Device (please complete the following information):**
 - Android Version:  13
 - App Version: 0.81.3

**Comment by gzuuus** [comment]
Update, it seems that my relays blew up for some reason and this was causing the application to misbehave, I have restored the relays again using a web client and so far the new version works fine. However the response of the app if the relays blow up for some reason is very unwieldy.

**Comment by vitorpamplona** [comment]
Humm.. Did all relays get deleted?

**Comment by gzuuus** [comment]
Yes, all the ones in my profile, I only had 3 of which only one I had configured before.
I have now restored them all and it works fine.

**Comment by vitorpamplona** [comment]
Closing this since it now we save relays in the local cache
